styles/_variables.scss

```diff
@@ -0,0 +1,5 @@
+// Copyright 2023 @paritytech/polkadot-dashboard-ui authors & contributors
+// SPDX-License-Identifier: Apache-2.0
+
+$page-width-threshold: 850px;
```

Comment:
These names are not intuitive. `page-width-small-threshold` and `page-width-medium-threshold` as discussed are okay for now.